### DISCUSSION TITLE
Proposed spec for RTO

### DISCUSSION THREAD

**Discussion by BLR-0118** [ONDC-Official/protocol-network-extension#48] [Announcements]
>Proposed spec for RTO is [here](https://docs.google.com/document/d/1dv6sEnYHU-fZxYP3-lZCpVwTS5ZP3MC38bSswAjWvs8/edit#) for comments

**Comment by BLR-0118** [comment]
@mherle @utkarsh-v @keshav-BFRS01751 @amitsaluja

**Comment by ONDC-Tech** [comment]
@VidyaGF
